package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "calendar",
-  "version": "1.0.23",
+  "version": "1.0.24",
```

Comment:
My bad, forget this, this is the result of the merge of master branch into development. Sorry.